elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/metadata/SQLTable.java

```diff
@@ -33,6 +40,32 @@
  */
 @EqualsAndHashCode(callSuper = true)
 public class SQLTable extends Table implements Queryable {
+
+    @Getter
+    private ConnectionDetails connectionDetails;
+
+    public SQLTable(Class<?> cls,
+                    EntityDictionary dictionary,
+                    ConnectionDetails defaultConnectionDetails,
+                    Map<String, ConnectionDetails> connectionDetailsMap) {
+        super(cls, dictionary);
+
+        String dbConnectionName = null;
+        Annotation annotation =
```

Comment:
I don't feel like the connectionDetailsMap is something this class should be concerned with.

The more we keep the metadata models like dumb pojos - the more flexibility we have down the road to do make certain modifications I see coming.  Let's not put any more annotation parsing stuff in them.

Reply:
updated.